app/src/main/res/values/dimens.xml

```diff
@@ -403,4 +403,14 @@
 
   <!-- Profile Edit -->
   <dimen name="profile_edit_delete_margin_bottom">12dp</dimen>
+
+
+  <!--  Concept Card Fragment-->
```

Comment:
Here and everywhere else write it as `<!-- ConceptCardFragment -->`

Notice one space at start and end of the text.